hazelcast/src/main/java/com/hazelcast/instance/impl/HazelcastBootstrap.java

```diff
@@ -153,10 +151,9 @@ public static synchronized void executeJar(@Nonnull Supplier<HazelcastInstance>
 
             URL jarUrl = new File(jarPath).toURI().toURL();
             // Close the URLClassLoader when finished
-            try (URLClassLoader classLoader = AccessController.doPrivileged(
-                    (PrivilegedAction<URLClassLoader>) () ->
-                            new URLClassLoader(new URL[]{jarUrl}, HazelcastBootstrap.class.getClassLoader())
-            )) {
+            try (URLClassLoader classLoader = URLClassLoader.newInstance(
```

Comment:
Currently URLClassLoader.newInstance() has AccessController.doPrivileged within itself. So it should not be a problem.

Also AccessController will be removed with the security manager packages. We should not be using AccessController
````
@Deprecated(since = "17", forRemoval = true) 
public final class AccessController
````